yjit/src/asm/x86_64/mod.rs

```diff
@@ -382,7 +390,7 @@ fn write_opcode(cb: &mut CodeBlock, opcode: u8, reg: X86Reg) {
 }
 
 /// Encode an RM instruction
-fn write_rm(cb: &mut CodeBlock, sz_pref: bool, rex_w: bool, r_opnd: X86Opnd, rm_opnd: X86Opnd, op_ext: u8, bytes: &[u8]) {
+fn write_rm(cb: &mut CodeBlock, sz_pref: bool, rex_w: bool, r_opnd: X86Opnd, rm_opnd: X86Opnd, op_ext: Option<u8>, bytes: &[u8]) {
```

Comment:
op_ext is actually always given, so we should pass it to write_rm with Some here instead of changing the callers of

Reply:
Not sure I'd prefer that. Having opt_ex change between Option<u8> and u8 and various places seems a bit confusing to me.

Reply:
It actually looks like we do call write_rm with no op_ext so I'd agree with Jimmy